frontend/server/src/Authorization.php

```diff
@@ -429,7 +435,13 @@ public static function canChooseCoderOrSchool(int $currentTimestamp): bool {
         return in_array($today, $availableDateToChooseCoder);
     }
 
-    public static function isCertificateGenerator(\OmegaUp\DAO\VO\Identities $identity): bool {
+    public static function isCertificateGenerator(
+        \OmegaUp\DAO\VO\Identities $identity,
+        ?int $role
+    ): bool {
+        if (!is_null($role) && $role === self::CERTIFICATE_GENERATOR_ROLE) {
```

Comment:
```suggestion
        if ($role === self::CERTIFICATE_GENERATOR_ROLE) {
```

esto implica que no es nulo. pero esto también significa que no se está validando nada si se pasa ese rol. eso es deseable? (me suena a que no, especialmente porque el usuario controla el valor de este parámetro y puede usarlo para brincarse la validación)